packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -512,16 +508,9 @@ SwipeableDrawer.propTypes = {
 };
 
 SwipeableDrawer.defaultProps = {
-  anchor: 'left',
-  disableBackdropTransition: false,
-  disableDiscovery: false,
   disableSwipeToOpen:
     typeof navigator !== 'undefined' && /iPad|iPhone|iPod/.test(navigator.userAgent),
-  hysteresis: 0.55,
-  minFlingVelocity: 400,
-  swipeAreaWidth: 20,
   transitionDuration: { enter: duration.enteringScreen, exit: duration.leavingScreen },
-  variant: 'temporary', // Mobile first.
 };
 
-export default withTheme(SwipeableDrawer);
+export default SwipeableDrawer;
```

Comment:
Remove most of the usages of `withTheme()`. It should help with perf, bundle size and debug noise.